manifests/d/dorssel/usbipd-win/1.2.0/dorssel.usbipd-win.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: dorssel.usbipd-win
+PackageVersion: 1.2.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+ProductCode: '{EA1D5623-E6A7-4E4A-9259-E3972226646F}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/dorssel/usbipd-win/releases/download/v1.2.0/usbipd-win_1.2.0.msi
+  InstallerSha256: 86F012512762A56B16C6A5CF9EFDEE4E9A9FFAC5E517D48212F8210A6CDB3E7C
```

Comment:
I'm OK with this change, but interestingly enough an earlier version was auto-corrected just the other way around by your YamlCreate.ps1 v2.0.0 script. Has that script been updated since?

See: https://github.com/microsoft/winget-pkgs/pull/29272/files

Reply:
Cannot accept suggestion, as it is only available for users with write permission on this repo.
Added a commit with the exact same change instead.

Reply:
Yes they wanted the ProductCode in the Installer so it would be less confusing for people that manually update the Yaml’s. (This got merged Friday)

The suggested changes should be towards your own fork, did you click the button below the suggested change?

Reply:
OK, I understand the ProductCode move.

About accepting the suggestion; that was not an option. Here is a screenshot of what it looks like on my end:
![image](https://user-images.githubusercontent.com/17404029/142247300-1fa28575-ced0-490b-8c47-bbcd46c7c75f.png)
The "commit suggestion" button is grayed out, and the popup help indicates that write permissions are lacking. What I don't know is whether that is about your rights or mine, or whether that is about your repo or my fork...